core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -104,6 +104,11 @@ class ConsumerBounceTest extends IntegrationTestHarness with Logging {
     consumer.subscribe(tp)
     consumer.seek(tp, 0)
 
+    // wait until all the followers have synced the last HW with leader
```

Comment:
@hachikuji the write was not lost, but just that the HW on the new leader was older than on the new leader, and that seekToEnd is guarded by the HW of the leader. I think it is irrelevant to the messages on the replicas.